libs/compute_api/src/spec.rs

```diff
@@ -79,6 +79,12 @@ pub struct ComputeSpec {
     // Stripe size for pageserver sharding, in pages
     #[serde(default)]
     pub shard_stripe_size: Option<usize>,
+
+    // When we are starting a new replica in hot standby mode,
+    // we need to know if the primary is running.
+    // This is used to determine if we should try to pull SLRU from primary
```

Comment:
This has nothing to do with SLRUs

Reply:
thank you for pointing out!
I confused running_xacts record with xacts SLURU